### DISCUSSION TITLE
During the Rust server installation process, I encounter an error. Please advise on what to do.

### DISCUSSION THREAD

**Discussion by kipishio** [GameServerManagers/LinuxGSM#4313] [Q&A]
=================================
LinuxGSM_

installing log dir: /home/kipishio/log...OK
installing LinuxGSM log dir: /home/kipishio/log/script...OK
creating LinuxGSM log: /home/kipishio/log/script/rustserver-script.log...OK
installing console log dir: /home/kipishio/log/console...OK
creating console log: /home/kipishio/log/console/rustserver-console.log...OK

Installing SteamCMD
=================================
Information! SteamCMD is already installed...OK

Installing Rust Server
=================================
**[ START ] Installing rustserver: SteamCMD
Redirecting stderr to '/home/kipishio/.steam/logs/stderr.txt'
ILocalize::AddFile() failed to load file "public/steambootstrapper_english.txt".
[  0%] Checking for available update...
KeyValues Error: LoadFromBuffer: missing {   (current key: '<!doctype') in file                                                    manifest [offset: 15]

../tier1/KeyValues.cpp (2925) : Assertion Failed: Error while parsing text KeyVa                                                   lues for resource manifest
[----] Verifying installation...
[  0%] Downloading Update...
[  0%] Checking for available update...
KeyValues Error: LoadFromBuffer: missing {   (current key: '<html><head><title>L                                                   oading...</title></head><body><script') in file manifest [offset: 62]

../tier1/KeyValues.cpp (2925) : Assertion Failed: Error while parsing text KeyVa                                                   lues for resource manifest
[----] !!! Fatal Error: Steamcmd needs to be online to update.   Please confirm                                                    your network connection and try again.
threadtools.cpp (3294) : Assertion Failed: Illegal termination of worker thread                                                    'Thread(0x0x56c1cb30/0x0xf79b9b'
Complete! Installing rustserver: SteamCMD
[ ERROR ] Installing rustserver: No appmanifest_258550.acf found
[ INFO ] Installing rustserver: Forcing update to correct issue
[ START ] Installing rustserver: SteamCMD
Redirecting stderr to '/home/kipishio/.steam/logs/stderr.txt'
ILocalize::AddFile() failed to load file "public/steambootstrapper_english.txt".
[  0%] Checking for available update...
KeyValues Error: LoadFromBuffer: missing {   (current key: '<html><head><title>L                                                   oading...</title></head><body><script') in file manifest [offset: 62]

../tier1/KeyValues.cpp (2925) : Assertion Failed: Error while parsing text KeyVa                                                   lues for resource manifest
[----] Verifying installation...
[  0%] Downloading Update...
[  0%] Checking for available update...
KeyValues Error: LoadFromBuffer: missing {   (current key: '<html><head><title>L                                                   oading...</title></head><body><script') in file manifest [offset: 62]

../tier1/KeyValues.cpp (2925) : Assertion Failed: Error while parsing text KeyVa                                                   lues for resource manifest
[----] !!! Fatal Error: Steamcmd needs to be online to update.   Please confirm                                                    your network connection and try again.
threadtools.cpp (3294) : Assertion Failed: Illegal termination of worker thread                                                    'Thread(0x0x581d6b30/0x0xf79d8b'
Complete! Installing rustserver: SteamCMD
[ FAIL ] Installing rustserver: Still no appmanifest_258550.acf found**
`

**Comment by unknown9449** [comment]
I get the same error, have you been able to work around this?
